lib/src/components/NxSmallVulnerabilityCounter/NxSmallVulnerabilityCounter.scss

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use "sass:math";
+@use "../../scss-shared/nx-text-helpers";
+
+@mixin counter-colors {
+  @each $severity in critical, high, medium, low, none {
+    &--#{$severity} {
+      background-color: var(--nx-color-vulnerability-#{$severity});
+      color: var(--nx-color-vulnerability-#{$severity}-text);
+      border: 1px solid var(--nx-color-vulnerability-#{$severity}-border);
+    }
+  }
+}
+
+.nx-small-vulnerability-counter {
+  $height: 25px;
+  position: relative;
+  border-radius: 2px;
+  font-size: var(--nx-font-size-s);
+  font-weight: 600;
+  height: $height;
+  line-height: $height;
+  padding-left: var(--nx-spacing-base);
+  padding-right: var(--nx-spacing-base);
+  text-align: center;
+
+  &__severity {
+    //invisible but still read by the screenreader
+    @include nx-text-helpers.visually-hidden;
+  }
+
+  @include counter-colors;
+
+  // Used to set the width of the counter based on its max text content
+  &__sizer {
+    display: block;
+    height: 0;
+    overflow: hidden;
+    visibility: hidden;
+  }
+
+  &--zero {
+    background-color: var(--nx-color-small-vulnerability-counter-zero-background);
+    color: var(--nx-color-small-vulnerability-counter-zero-text);
+    border: 1px solid var(--nx-color-small-vulnerability-counter-zero-border);
+  }
+}
+
+.nx-small-vulnerability-counter-container {
+  display: inline-flex;
+  gap: var(--nx-spacing-2x);
```

Comment:
Should be 4 not 8

Reply:
> Should be 4 not 8

Updated here: [02a4350](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/02a4350e0be23e4f67c8161556ccb26d7a442eb8)

Reply:
> > Should be 4 not 8
> 
> Updated here: [02a4350](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/02a4350e0be23e4f67c8161556ccb26d7a442eb8)

It includes the margin, but let me triple check with design

Reply:
Changed the margins back to 8 here (not gap) after confirming with design
[615b1a1](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/615b1a1ad8bce668ed863f1d15573e633ea73186)